src/Business/Detection/Common/Models/PDetection.cs

```diff
@@ -0,0 +1,18 @@
+namespace Business.Detection.Common.Models
+{
+    using System;
+    using Business.Pagination;
+
+    public class PDetection
+    {
+        public int Id { get; set; }
+
+        public int Score { get; set; }
+
+        public string Class { get; set; }
+
+        public DateTime Timestamp { get; set; }
+
+        public PPagination Pagination { get; set; } = new PPagination();
+    }
+}
```

Comment:
As far as we are gonna use this mode for fetching only, won't be better to move it under the `Detection.Fetching.Models` namespace?

Reply:
Done